commands/countlinetotal/index.js

```diff
@@ -26,15 +26,25 @@ module.exports = {
 			.limit(1)
 			.single()
 		);
-	
-		const days = (sb.Date.now() - history.Executed) / 864.0e5;
-		const originalSize = Number(history.Result.match(/([\d.]+) GB of space/)[1]);
+
+		let historyText = "";
 		const currentSize = sb.Utils.round(data.Bytes / (10 ** 9), 3);
-	
-		const rate = sb.Utils.round((currentSize - originalSize) / days, 3);
-		const fillDate = new sb.Date().addDays((220 - currentSize) / rate); // 238 GB minus an estimate of ~18GB of other stuff
-		const megabytesPerHour = sb.Utils.round(rate * 1024 / 24, 3);
-	
+		
+		if(history) {
+			const days = (sb.Date.now() - history.Executed) / 864.0e5;
+			const originalSize = Number(history.Result.match(/([\d.]+) GB of space/)[1]);
+			const rate = sb.Utils.round((currentSize - originalSize) / days, 3);
+			const megabytesPerHour = sb.Utils.round(rate * 1024 / 24, 3);
+			const fillDate = new sb.Date().addDays((220 - currentSize) / rate); // 238 GB minus an estimate of ~18GB of other stuff
+			historyText = (megabytesPerHour === 0) ? 
```

Comment:
split this ternary into two strings - the `MB/s` string is used both times, whereas the `fillup` one isn't
afterwards, combine them both into `historyText`